pce:
  planning_groups:
    - manipulator
    - arm
    - panda_arm
  
  panda_arm:
    pce_planner:
      num_samples: 3000
      num_iterations: 20
      eta: 0.99
      temperature: 2.0
      convergence_threshold: 0.0001
      num_discretization: 20
      total_time: 5.0
      node_collision_radius: 0.20

    collision_clearance: 0.15
    collision_threshold: 0.15